pilight/protocol_list.json

```diff
@@ -0,0 +1,2638 @@
+{
```

Comment:
This is generated from development? We could incorporate a protocol version (e.g. release 7). But it is not that important.